web_interface/blueprints/api_v3.py

```
@@ -2433,6 +2479,19 @@ def reconcile_plugin_state():
             status_code=500
         )
 
+@api_v3.route('/plugins/reconciliation-status', methods=['GET'])
+def get_reconciliation_status():
+    """Return the result of the last startup reconciliation from /tmp status file."""
+    _recon_path = "/tmp/ledmatrix_reconciliation.json"
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/ledmatrix_reconciliation.json"

(S108)